src/main/java/View/searchOutputView.java

```diff
@@ -12,37 +10,32 @@
 import java.awt.event.ActionListener;
 
 public class searchOutputView extends JFrame implements ActionListener {
-
-
+    private playlistView playlistView;
     private String searchText;
     private int FONTSIZE = 10;
     private int WIDTH = 640;
     private int HEIGHT = 640;
-
     private JFrame jframe;
     private JPanel panel;
     private JScrollPane scrollPane;
     private JLabel title;
     private JButton homeButton;
     private Font font;
-
-
     private JTable table;
     private BorderLayout layout;
     private JComboBox comboBox;
     private String[][] data;
     private String[] ids;
-
     private searchViewModel searchViewModel;
 
-
-    public searchOutputView(String searchText){
-        this.initialiseValues(searchText);
+    public searchOutputView(String searchText, playlistView playlistView){
+        this.initialiseValues(searchText, playlistView);
         this.setUpTable();
-        this.initialiseFrame();
+        this.initializeFrame();
     }
 
-    public void initialiseValues(String searchText){
+    public void initialiseValues(String searchText, playlistView playlistView){
+        this.playlistView = playlistView;
```

Comment:
Is this necessary? I don't think it will be needed once we get the logged in users working, since we can just retrieve all the information again. I believe raf's pull request will deal with this. It will cause conflicts if we merge your pull request that has playlistView as a parameter and then try to merge rafs after which doesn't have it. I think it might just be easier to delete it from yours before we merge this